oxyno-zeta/s3-proxy
# fix(authentication): Fix invalid redirect uri

## Issue/Feature

Close #96 

## Additional Information

## Verification Steps

1. Configure your Keycloak with a full redirect URI without any star
2. Go on a page, for example: http://localhost:8080/v2/test without being connected
3. The redirect to Keycloak will be done
4. See no error

## Checklist:
